src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const listElement = document.querySelector('ul');
+const employeesArr = [...listElement.children];
+
+listElement.innerHTML = '';
+
+sortWorkers(employeesArr);
+getEmployees(employeesArr);
+
+function sortWorkers(arr) {
+  const sortedWorkers = arr.sort((employee1, employee2) => {
+    const salary1 = +employee1.dataset.salary.replace(/[$,]/g, '');
+    const salary2 = +employee2.dataset.salary.replace(/[$,]/g, '');
```

Comment:
The task suggests writing a helper function for converting salary strings to numbers. Consider extracting this logic into a separate function for clarity and reusability.